### DISCUSSION TITLE
feature request: expose packageRules via environment variable

### DISCUSSION THREAD

**Discussion by nabeelsaabna** [renovatebot/renovate#24049] [Suggest an Idea]
### Tell us more.

For a package version management from an external system it's required that `packageRules` are accessible from a config outside of the repository

**Comment by secustor** [comment]
Are you aware that you can set `packageRules` in the global config? 
[Which you can set via environment variables too?](https://docs.renovatebot.com/getting-started/running/#global-config)

**Reply by nabeelsaabna** [reply]
Hi,
I'm aware of that, it's due to language limitation and ease of use.
I believe the only reason that `packageRules` is not exposed as environment variable is the lack of use case as a global config option

**Comment by jamietanna** [comment]
This is now available in `RENOVATE_PACKAGE_RULES`
